ServerCore/Pages/Pieces/InternMeta.cshtml

```diff
@@ -0,0 +1,27 @@
+﻿@page "/{eventId}/{eventRole}/Teams/{teamId}/InternMeta/{puzzleId}"
+@model ServerCore.Pages.Pieces.InternMetaModel
+
+@{
+    ViewData["Title"] = "Intern Puzzleday Meta";
+}
+
+<h2>Metapuzzle Pieces Earned</h2>
+
+<table class="table">
+    <thead>
+        <tr>
+            <th>
+                @Html.DisplayNameFor(model => model.EarnedPieces[0].Contents)
+            </th>
+        </tr>
+    </thead>
+    <tbody>
+@foreach (var item in Model.EarnedPieces) {
+        <tr>
+            <td>
+                @Html.DisplayFor(modelItem => item.Contents)
```

Comment:
A useful future frill would be the ability to have the pieces contain Markdown that gets rendered by this page.  (Alternately, you could just let pieces contain HTML that gets inserted raw into the page, but this is dangerous if you don't trust authors to sanitize their piece content.)

Reply:
Agree but I'm not ready to look up how to render markdown from a Razor page yet.